pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -33,28 +36,38 @@ From the list of healthy peers that you retrieve from peer node provider, choose
-
-### State Sync
-You can use **state sync**, a set of configuration settings that allow your node to retrieve a snapshot from the network. If your node falls out of sync, it queries a state sync node for a verified, recent snapshot of the application state. This speeds up the syncing process because you avoid replaying the entire history of the network, instead starting from the network's most recent application state snapshot. To use state sync to back up your full node, follow the instructions for your deployment from a [State Sync](../infrastructure_providers-network/resources.mdx#state-sync-service) service.
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-## Optimize Pruning Settings
+### Configure a Pruning Strategy
 In general, dYdX recommends the following pruning setting, configured in your `app.toml` file:
 
 ```bash
 # app.toml
 pruning = "everything" # 2 latest states will be kept; pruning at 10 block intervals
 ```
 
-However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
\ No newline at end of file
+However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
+
+## Prepare to Restore Your Node
+Prepare to restore your node quickly in case it falls out of sync. 
+
+Your full node can fall out of sync with the rest of the network for a variety of reasons, including a bad software upgrade, unexpected node crashes, or human operational error. To re-sync with the network, your full node must replay the history of the network.
+
+You can speed up the re-syncing process significantly by providing your node with a snapshot. A snapshot contains a compressed copy of the application state at the time the snapshot was taken. If your node falls out of sync, a snapshot allows it to recover to that saved state before replaying the rest of the history of the network, saving time.
+
+### Save a Snapshot on Your System
+You can use a snapshot saved on the system on which your node runs to restore your node.
+
+To use a snapshot to back up your full node, install a snapshot for your deployment from a [Snapshot Service](../infrastructure_providers-network/resources.mdx#snapshot-service).
+
+> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed.
+
+### Configure Your Node's State Sync
```

Comment:
I think this is more automated than manually downloading snapshots, so would it make sense to recommend this over the manual snapshot route? if so, can we move this section to be above the other option?

Reply:
That makes a lot of sense. Also, from our discussion, it seems like practically you'd want to do a little more than save one snapshot (you'd want your own process to update the saved snapshot). I'll reorder like you suggested, and longer term I wonder if it makes sense to take this out. Could seem like were recommending saving one snapshot. I think if we remove the snapshot section it should be in another PR that also addresses the step in Set Up a Node